directory/hatch.toml

```diff
@@ -0,0 +1,11 @@
+[env.collectors.datadog-checks]
+
+[envs.default]
+dependencies = [
+  "virtualenv==20.17.1",
```

Comment:
Might be, but is this actually used?

Reply:
don't think so, I just added it because it was there. I removed it!